test/ex_plasma/utxo_test.exs

```diff
@@ -4,6 +4,24 @@ defmodule ExPlasma.UtxoTest do
 
   alias ExPlasma.Utxo
 
+  describe "new/1" do
+    test "does not allow amount to be zero" do
+      assert {:error, {:amount, :cannot_be_zero}} == Utxo.new(%Utxo{amount: 0})
+    end
+
+    test "does not allow output guard / owner to be zero" do
+      assert {:error, {:owner, :cannot_be_zero}} == Utxo.new(%Utxo{owner: <<0::160>>})
+    end
+
+    test "does not allow blknum to exceed maximum" do
```

Comment:
will an attempt to decode an input (utxo position) containing a too large blknum also fail (what #21 specifically raised)? if yes, do we want a test for this? (possibly similar remark to other conditions picked up when `new/1` is decoding)

Reply:
I've added code to do the bi-directional validation And test cases in 2df6d47.